test/WebJobs.Script.Tests/Configuration/ExtensionBundleOptionsSetupTest.cs

```diff
@@ -171,5 +257,22 @@ private IConfiguration BuildHostJsonConfiguration(IEnvironment environment = nul
 
             return configurationBuilder.Build();
         }
+
+        private TestEnvironment GetAppServiceEnvironment()
```

Comment:
Can we have a better name for this method? The current name doesn't clearly describe what it does.